src/main/java/org/broadinstitute/hellbender/engine/filters/MarkDuplicatesReadFilter.java

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.tools.spark.transforms.markduplicates.MarkDuplicatesSparkUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.Serializable;
+
+/**
+ * Filters out reads marked as duplicates.
+ */
+public final class MarkDuplicatesReadFilter extends ReadFilter implements Serializable {
```

Comment:
It seems confusing to me to have `MarkDuplicatesReadFilter` here and `ReadFilterLibrary.NotDuplicateReadFilter` in the same package. It might be more consistent to call this `NotOpticalDuplicateReadFilter` and chain it with `NotDuplicateReadFilter` if desired. Did @lbergelson already comment on this?